types/html-to-text/index.d.ts

```diff
@@ -196,6 +147,11 @@ export interface LimitsOptions {
      *
      * No limit if undefined.
      */
+    maxBaseElements?: number | undefined;
+    /**
+     * Maximum number of child nodes of a single node to be added to the
+     * output. Unlimited if undefined.
+     */
```

Comment:
Looks like a merge failure or something like that. Looks like `maxBaseElements` gets the old doc string of `maxChildNodes`.

Reply:
Updated.  Not sure why I missed that.  @KillyMXI Thanks for the review.